### ISSUE TITLE
Bug: [Accessibility Issue] Switches are not visible in light mode

### ISSUE THREAD

**Issue by SGI-CAPP-AT2** [CCExtractor/ultimate_alarm_clock#612]
### Description

When switched to light mode the Switches in settings are not visible 

### Screenshots

<img src="https://github.com/user-attachments/assets/2547d90a-49d9-412b-b39b-acc5c4dcaf6c" width=200/>

**Comment by SGI-CAPP-AT2** [comment]
I would like to work on this

**Comment by ganidande905** [comment]
Hi there,

I noticed that this issue is currently unassigned. I've fixed the bug and created a pull request addressing it. I kindly request this issue to be assigned to me as I have already worked on it.

Looking forward to your review of the PR. Thank you!

**Comment by SGI-CAPP-AT2** [comment]
@ganidande905 Thanks for solving, 
But I was working on the same.

please always check for updates in issue discussions if someone is working or not and notify that you're working on it.

**Comment by ganidande905** [comment]
Thank you for your message @SGI-CAPP-AT2. You're more than welcome to create a pull request as well! Ultimately, it’s up to the organization to decide whom to assign the issue to and which solution aligns best with their vision.

Let’s both contribute our best and leave the final decision to the maintainers. Collaboration always leads to better results!
